### ISSUE TITLE
ReferenceError: cacheDirectory is not defined

### ISSUE THREAD

**Issue by darul75** [babel/babel-loader#75]
Hi,

currently on window, I have the following trace but can not figure what is wrong

```
ser.js 831 bytes {0} [built]
  [269] (webpack)-dev-server/~/socket.io-client/~/has-binary/index.js 1.08 kB {0
} [built]
  [270] (webpack)-dev-server/~/socket.io-client/~/has-binary/~/isarray/index.js
120 bytes {0} [built]
  [271] (webpack)-dev-server/~/socket.io-client/~/object-component/index.js 1.18
 kB {0} [built]
  [272] (webpack)-dev-server/~/socket.io-client/~/parseuri/index.js 690 bytes {0
} [built]
  [273] (webpack)-dev-server/~/socket.io-client/~/socket.io-parser/binary.js 3.8
4 kB {0} [built]
  [274] (webpack)-dev-server/~/socket.io-client/~/socket.io-parser/~/json3/lib/j
son3.js 40.1 kB {0} [built]
  [275] (webpack)-dev-server/~/socket.io-client/~/to-array/index.js 216 bytes {0
} [built]
  [276] (webpack)-dev-server/~/strip-ansi/index.js 161 bytes {0} [built]
  [277] (webpack)-dev-server/~/strip-ansi/~/ansi-regex/index.js 145 bytes {0} [b
uilt]
  [278] (webpack)/buildin/amd-options.js 43 bytes {0} [built]
  [279] (webpack)/hot/log-apply-result.js 813 bytes {0} [built]
  [280] ./~/whatwg-fetch/fetch.js 8.54 kB {0} [built]
chunk    {1} app.js (app) 52 bytes {0} [rendered]
    [0] multi app 52 bytes {1} [built] [1 error]

ERROR in ./app/app.js
Module build failed: ReferenceError: cacheDirectory is not defined
    at Object.module.exports (C:/JUL/DEV/github/web-react/node_modules/babel-loa
der/index.js:42:17)
 @ multi app
webpack: bundle is now VALID.`
```

webpack instruction is the following:

```
{ test: /\.js?$/, loaders: ['react-hot', 'babel'], exclude: /node_modules/ },`
```

**Comment by thealjey** [comment]
duplicate of #74

**Comment by Couto** [comment]
I just closed to concentrate the discussion on #74 
The issue is valid and the patch is ready
